applications/DEMApplication/custom_constitutive/DEM_D_Linear_HighStiffness_CL.h

```diff
@@ -23,11 +23,16 @@ namespace Kratos {
 
         std::unique_ptr<DEMDiscontinuumConstitutiveLaw> CloneUnique() override;
 
+        void Check(Properties::Pointer pProp) const override;
+
         void InitializeContact(SphericParticle* const element1, SphericParticle* const element2, const double indentation) override;
 
         void InitializeContactWithFEM(SphericParticle* const element, Condition* const wall, const double indentation, const double ini_delta = 0.0) override;
 
-    };
+    protected:
 
+        Properties::Pointer mpProperties;
```

Comment:
Discontinuum walls do not have member variables. Are we sure we need this one? We can find the properties of this contact by doing:
`Properties& properties_of_this_contact = element1->GetProperties().GetSubProperties(element2->GetProperties().Id());`

Reply:
Thanks for the comment. `Properties::Pointer mpProperties;` has been removed. For ball-ball and ball-wall contacts, the `Properties& properties_of_this_contact = element1->GetProperties().GetSubProperties(element2->GetProperties().Id());` is used.